.github/workflows/action.yml

```diff
@@ -10,7 +10,7 @@ permissions:
 
 jobs:
   quick-tests:
-    runs-on: ubuntu-latest
+    runs-on: raspbian-armv7-kernel-5.10.33
```

Comment:
This would add a blocker for any newer contributors to test their changes on their repos. ( they'll not have this runner)
Also a full building from zero on raspberry takes 20~25 minutes. Would suggest to keep it on gh's CI, as it can deliver linter & format checks much faster and preservers self-hosted-armv7 runners ( we only have 2, and shared btwn ping-rs, navigator-lib, ping-viewer repos)

Reply:
Done.